disco_aws_automation/disco_aws.py

```diff
@@ -363,14 +366,17 @@ def provision(self, ami, hostclass=None,
             tags={"hostclass": hostclass,
                   "owner": user_data["owner"],
                   "environment": self.environment_name,
-                  "chaos": chaos},
-            load_balancers=[elb['LoadBalancerName']] if elb else []
+                  "chaos": chaos,
+                  "is_testing": "1" if testing else "0"},
```

Comment:
weird... then maybe `is_truthy()` should be updated to also handle boolean and not only strings... neways, that's not in scope